app/javascript/components/ansible-credentials-form/ansible-credential-form.schema.js

```diff
@@ -0,0 +1,37 @@
+import { componentTypes } from '@@ddf';
+
+const createSchema = (fields, promise, edit, loadSchema) => ({
+  fields: [
+    {
+      component: componentTypes.TEXT_FIELD,
+      id: 'name',
+      name: 'name',
+      label: __('Name'),
+      maxLength: 128,
+    },
```

Comment:
I'm pretty sure that in the angular form this field is set to required, in DDF you need these two attributes:
```
isRequired: true,
validate: [{ type: validatorTypes.REQUIRED }]
```

and you need to import the `validatorTypes` the same way as you do `componentTypes` from `@@ddf`